src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortBamSparkIntegrationTest.java

```diff
@@ -1,15 +1,60 @@
 package org.broadinstitute.hellbender.tools.spark.pipelines;
 
+import htsjdk.samtools.ValidationStringency;
 import org.broadinstitute.hellbender.CommandLineProgramTest;
 import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
 import org.broadinstitute.hellbender.utils.test.SamAssertionUtils;
 import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.File;
 
 public final class SortBamSparkIntegrationTest extends CommandLineProgramTest {
+    @DataProvider(name="sortbams")
+    public Object[][] sortBAMData() {
+        return new Object[][] {
+                {"count_reads.bam", "count_reads_sorted.bam", null, ".bam", "coordinate"},
+                {"count_reads.bam", "count_reads_sorted.bam", "count_reads.fasta", ".cram", "coordinate"},
 
+                //SortBamSpark is missing SORT_ORDER parameter  https://github.com/broadinstitute/gatk/issues/1260
+//                {"count_reads.bam", "count_reads.bam", null, ".bam", "queryname"},
+//                {"count_reads.cram", "count_reads.cram", "count_reads.fasta", ".cram", "queryname"},
+
+                {"count_reads.cram", "count_reads_sorted.cram", "count_reads.fasta", ".cram", "coordinate"},
+                {"count_reads.cram", "count_reads_sorted.cram", "count_reads.fasta", ".bam", "coordinate"},
+        };
+    }
+
+    @Test(dataProvider="sortbams")
+    public void testSortBAMs(
+            final String inputFileName,
+            final String expectedOutputFileName,
+            final String referenceFileName,
+            final String outputExtension,
+            final String sortOrderName) throws Exception
+    {
+        final File inputBam = new File(getTestDataDir(), inputFileName);
+        final File expectedBam = new File(getTestDataDir(), expectedOutputFileName);
+        final File outputBam = createTempFile("sort_sam", "outputExtension");
```

Comment:
This line of code has the  same bug I fixed on Friday in the non-spark version, which is "outputExtension" is in quotes when it shouldn't be, so its a literal.